pkg/command/service/clean_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright © 2020 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package service
+
+import (
+	"testing"
+
+	"gotest.tools/assert"
+	k8sfake "k8s.io/client-go/kubernetes/fake"
+	"knative.dev/kperf/pkg"
+	"knative.dev/kperf/pkg/testutil"
+)
+
+func TestNewServiceCleanCommand(t *testing.T) {
+	t.Run("incompleted args for service clean", func(t *testing.T) {
+		client := k8sfake.NewSimpleClientset()
+		p := &pkg.PerfParams{
+			ClientSet: client,
+		}
+		cmd := NewServiceCleanCommand(p)
+
+		_, err := testutil.ExecuteCommand(cmd)
+		assert.ErrorContains(t, err, "both ns and nsPrefix are empty")
+	})
+}
```

Comment:
@gyohuangxin 
I think we need at least one positive test case for clean command, for e.g.

- fake a service
- ExecuteCommand
- check faked service is deleted